app/services/trainees/create_from_hesa.rb

```diff
@@ -138,10 +138,10 @@ def lead_partner_attributes
 
       if hesa_trainee[:ukprn].in?(LEAD_PARTNER_TO_ACCREDITED_PROVIDER_MAPPING.keys)
         attrs.merge!(lead_partner: LeadPartner.find_by(ukprn: hesa_trainee[:ukprn]), lead_partner_not_applicable: false)
-      elsif hesa_trainee[:lead_school_urn].in?(NOT_APPLICABLE_SCHOOL_URNS)
+      elsif hesa_trainee[:lead_partner_urn].in?(NOT_APPLICABLE_SCHOOL_URNS)
```

Comment:
Just a note about the timing of changes to this HESA process and the HESA testing that's currently in progress. Nothing to action, but wanted us to be aware of it.